core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -346,7 +370,7 @@ Optional<BlockMeta> removeBlockInternal(long sessionId, long blockId, long timeo
     }
 
     try (BlockLock lock = optionalLock.get();
-        LockResource r = new LockResource(mMetadataWriteLock)) {
+         LockResource r = new LockResource(mMetadataWriteLock)) {
```

Comment:
We use 4 spaces indent?
But there are 5 spaces here.

Reply:
Here line 373 and `BlockLock lock = optionalLock.get();` in line 372  are in the same statement. So actually line 373 need to have 2 spaces after the keywords `try` in line 372.